src/ostree.h

```diff
@@ -24,10 +24,12 @@ class OstreeManager : public PackageManagerInterface {
   boost::shared_ptr<OstreeDeployment> getStagedDeployment();
   static boost::shared_ptr<OstreeSysroot> LoadSysroot(const boost::filesystem::path &path);
   static bool addRemote(OstreeRepo *repo, const std::string &url, const KeyManager &keys);
-  static data::InstallOutcome pull(const Config &config, const KeyManager &keys, const std::string &refhash);
+  statuc data::InstallOutcome OstreeManager::find_or_pull(const Config &config, const KeyManager &keys,
```

Comment:
`statuc`?

Reply:
The `OstreeManager::` is redundant, too.

Reply:
Sorry, compiled with BUILD_OSTREE=OFF before pushing.